test/server/cards/03_TWI/units/OutspokenRepresentative.spec.ts

```diff
@@ -0,0 +1,33 @@
+describe('Outspoken Representative', function () {
+    integration(function (contextRef) {
+        it('should gain Sentinel and create a Clone Trooper token when defeated', function () {
+            contextRef.setupTest({
+                phase: 'action',
+                player1: {
+                    hand: ['clone-heavy-gunner'],
+                    groundArena: ['outspoken-representative'],
+                },
+                player2: {
+                    groundArena: ['specforce-soldier', 'volunteer-soldier'],
+                    hasInitiative: true,
+                },
+            });
+            const { context } = contextRef;
+
+            // Check if sentinel is not yet active
+            context.player2.clickCard(context.specforceSoldier);
+            expect(context.player2).toBeAbleToSelectExactly([context.outspokenRepresentative, context.p1Base]);
+            context.player2.clickCard(context.outspokenRepresentative);
+
+            // Check if sentinel is now active
+            context.player1.clickCard(context.cloneHeavyGunner);
+            context.player2.clickCard(context.volunteerSoldier);
+            expect(context.player2).toBeAbleToSelectExactly([context.outspokenRepresentative]);
+
+            // Check if clone token is created when defeated
+            context.player2.clickCard(context.outspokenRepresentative);
+            const cloneTroopers = context.player1.findCardsByName('clone-trooper');
+            expect(cloneTroopers.length).toBe(1);
```

Comment:
Also need to check the zone. Since tokens are created in the `outsideTheGameZone` we have to confirm they were correctly put into play in the ground arena. Same for the battle droid check in Wartime Trade Official